keylime/config.py

```diff
@@ -180,31 +183,27 @@ def get_config(component: str) -> RawConfigParser:
             if os.path.exists(CONFIG_ENV[component]):
                 if os.path.isfile(CONFIG_ENV[component]):
                     config_files = _config[component].read(CONFIG_ENV[component])
-                    print(f"Reading configuration from {config_files}")
+                    log_msg = f"Reading configuration from {config_files}"
+                    base_logger.debug(log_msg)
                     return _config[component]
 
-                print(
-                    f"Configuration file {CONFIG_ENV[component]} for"
-                    f"{component} set through environment variable is "
-                    "not a file, falling back to installed configuration"
-                )
+                log_msg = f"Configuration file {CONFIG_ENV[component]} for {component} set through environment variable is not a file, falling back to installed configuration"
+                base_logger.debug(log_msg)
```

Comment:
same (to avoid repetition I will skip the rest)

Reply:
same